README.md

```
@@ -1,102 +1,99 @@
+```bash
+# リポジトリをクローン
+git clone https://github.com/SphereStacking/dotfiles.git ~/dotfiles
+cd ~/dotfiles
+
+# セットアップスクリプトを実行
+./bootstrap.sh
+```
+
+## ファイル構成
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)